swap_meet/vendor.py

```diff
@@ -1,2 +1,95 @@
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self,item):
+        inventory = self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            inventory = self.inventory.remove(item)
+            return item
+        return False
```

Comment:
This is a small tweak, but it can often be a good idea to follow a pattern of starting with the negative guard clause as a conditional and then treating the rest of our function as the intended operation of the function. With that in mind, I would swap these checks so that we can have the "happy case" outside the conditional entirely!

```suggestion
    def remove(self,item):
        if item not in self.inventory:
            return False
            
        inventory = self.inventory.remove(item)
        return item
```